tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -439,7 +426,14 @@ $(shell find tensorflow/lite/ -type d \( -path tensorflow/lite/experimental -o -
 MICROLITE_CC_BASE_SRCS := \
 $(wildcard tensorflow/lite/micro/*.cc) \
 $(wildcard tensorflow/lite/micro/memory_planner/*.cc) \
-$(TFL_CC_SRCS)
```

Comment:
we need to keep this variable around. Its unclear to me why this would cause trouble.